packages/scipy/patches/0011-When-forward-declaring-print_soln-give-it-the-correc.patch

```diff
@@ -0,0 +1,70 @@
+From bb4e0d92ee2ba91ac5defd36ca5781433de25688 Mon Sep 17 00:00:00 2001
+From: Hood Chatham <roberthoodchatham@gmail.com>
+Date: Fri, 10 Mar 2023 14:47:00 +0000
+Subject: [PATCH 11/11] When forward declaring print_soln, give it the correct
+ signature
+
```

Comment:
Maybe add a comment saying it's fixed by https://github.com/scipy/scipy/pull/18124

Reply:
Unresolving as I think it still needs the above  comment in the patch description.

Reply:
I actually have added the info in the scipy meta.yaml instead https://github.com/pyodide/pyodide/pull/3331/files#diff-d6d5fb2d234d6c6bafa1cff3848a6f49e87a62111e50baab21b4e954ae2b85be